package.json

```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "delay": "^2.0.0",
     "chai": "^4.0.0",
-    "marathon-event-bus-mock": "^0.1.2",
+    "marathon-event-bus-mock": "git+https://github.com/idanto/marathon-event-bus-mock.git",
```

Comment:
Please use the version `0.1.3` of the [tobilg/marathon-event-bus-mock](https://github.com/tobilg/marathon-event-bus-mock) module.

Reply:
done